pkg/sdk/sequences_impl_gen.go

```diff
@@ -0,0 +1,143 @@
+package sdk
+
+import (
+	"context"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+)
+
+var _ Sequences = (*sequences)(nil)
+
+type sequences struct {
+	client *Client
+}
+
+func (v *sequences) Create(ctx context.Context, request *CreateSequenceRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *sequences) Alter(ctx context.Context, request *AlterSequenceRequest) error {
+	opts := request.toOpts()
+	return validateAndExec(v.client, ctx, opts)
+}
+
+func (v *sequences) Show(ctx context.Context, request *ShowSequenceRequest) ([]Sequence, error) {
+	opts := request.toOpts()
+	dbRows, err := validateAndQuery[sequenceRow](v.client, ctx, opts)
+	if err != nil {
+		return nil, err
+	}
+	resultList := convertRows[sequenceRow, Sequence](dbRows)
+	return resultList, nil
+}
+
+func (v *sequences) ShowByID(ctx context.Context, id SchemaObjectIdentifier) (*Sequence, error) {
+	request := NewShowSequenceRequest().WithIn(&In{Database: NewAccountObjectIdentifier(id.DatabaseName())}).WithLike(&Like{String(id.Name())})
```

Comment:
shouldn't we narrow it down to 'in schema' instead of 'in database' ?

Reply:
fixed